package/gluon-web-admin/files/lib/gluon/config-mode/view/admin/info.html

```diff
@@ -6,14 +6,15 @@
 	local sysconfig = require 'gluon.sysconfig'
 	local platform = require 'gluon.platform'
 	local util = require "gluon.util"
+	local vpn = require 'gluon.mesh-vpn'
```

Comment:
I suppose this will break things, if `gluon-mesh-vpn-core` is not baked into the firmware.

Reply:
IMHO depending on the core VPN package is better, given it is only pulling in some minified scripts.

Reply:
It already depends on `gluon-config-mode-core`

Reply:
As of now, this is not a good idea since `gluon-mesh-vpn-core` checks the site.conf with its `check_site.lua`. If we add this dependency, this would mean that even if you want to build an image without vpn, you would have to define e.g. `mesh_vpn.mtu` in your site.conf. (For reference: [Link](https://github.com/freifunk-gluon/gluon/blob/master/package/gluon-mesh-vpn-core/check_site.lua#L2))

Reply:
`gluon-config-mode-core` != `gluon-mesh-vpn-core`